guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -14,6 +15,12 @@ If your {ProjectServer} uses an external database, the database will be upgraded
 # runuser -l postgres -c \
 "psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\""
 ----
+. Verify that {Project} can reach the new database:
```

Comment:
This part of the procedure runs on `{ProjectServer}` but the `runuser` command above runs on the external DB host. That is confusing.